common/app/com/commercetools/sunrise/framework/reverserouters/myaccount/authentication/AuthenticationLinksControllerComponent.java

```diff
@@ -26,5 +26,7 @@ protected void addLinksToPage(final PageMeta meta, final AuthenticationReverseRo
         meta.addHalLink(reverseRouter.logOutProcessCall(), "logOut");
         meta.addHalLink(reverseRouter.logInPageCall(), "signUp");
         meta.addHalLink(reverseRouter.signUpProcessCall(), "signUpSubmit");
+        meta.addHalLink(reverseRouter.changePasswordPageCall(), "changePassword");
```

Comment:
Additionally I would suggest to introduce a separate `ChangePasswordLinksControllerComponent` class and a corresponding `ChangePasswordReverseRouter` interface with a `DefaultChangePasswordReverseRouter` class.